addon/templates/components/bootstrap-data-table.hbs

```diff
@@ -0,0 +1,45 @@
+<div class="bootstrap-data-table-component" style="padding: 5px">
+  {{#if title}}
+    <div class="row">
+      <div class="col-md-12">
```

Comment:
The title class `col-md-12` may be too specific on the width here. What is the advantage of having the row div?